src/fn/watch-project.js

```diff
@@ -0,0 +1,177 @@
+const environment = require('../lib/environment');
+const process = require('process');
+const path = require('path');
+const { error, info } = require('../lib/log');
+const Queue = require('queue-promise');
+const watcher = require('@parcel/watcher');
+const convertForms = require('../lib/convert-forms');
+const uploadForms = require('../lib/upload-forms');
+const convertContactForm = require('./convert-contact-forms').execute;
+const { uploadAppSettings } = require('./upload-app-settings');
+const compileAppSettings = require('./compile-app-settings').execute;
+const uploadCustomTranslations = require('./upload-custom-translations').execute;
+const uploadResources = require('./upload-resources').execute;
+
+const formXLSRegex = /^[a-zA-Z0-9_-]+\.xlsx$/;
```

Comment:
I think these regexes are a bit risky. For example, the code we use to fetch which files to convert in `conver-forms` only checks for the extension. I think we should find a way to reuse that code here as well, so there are no differences. 
Another idea, similar to the other comment I added, would be for the "fn" itself to export the extension to watch.